src/styles/blocks/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$general-color: #060b35;
+$second-color: #00acdc;
+$third-color: #616070;
```

Comment:
Let's don't mix numbers in variable names

Reply:
I will remember
I won't correct it here, because I haven't thought of a name. 
I think it will be more clear in the projects or will be provided by the designer.